src/main/java/com/microsoft/azure/spring/data/documentdb/support/IllegalCollectionException.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.data.documentdb.support;
+
+import org.springframework.dao.DataAccessException;
+import org.springframework.lang.Nullable;
+
+public class IllegalCollectionException extends DataAccessException {
```

Comment:
pack all exception class under .exception namespace?

Reply:
we may add more exceptions in future, it will be nice to have a exception namespace and folder

Reply:
changed package name to `.exception`